src/VirtualClient/VirtualClient.Main/profiles/PERF-MYSQL-SYSBENCH-OLTP.json

```diff
@@ -0,0 +1,2021 @@
+        "DatabaseName": "sbtest", 
+        "Role": "Client",
+        "Threads": "32",
+        "NumTables": "4",
+        "RecordCount": "100000",
+        "DurationSecs": "1800",
+        "Workload": "oltp_insert",
+        "PackageName": "sysbench"
+      }
+    },
+    {
+      "Type": "SysbenchOLTPClientExecutor",
+      "Parameters": {
+        "Scenario": "oltp_insert_T64_TB4_REC100000",
+        "DatabaseName": "sbtest", 
+        "Role": "Client",
+        "Threads": "64",
+        "NumTables": "4",
+        "RecordCount": "100000",
+        "DurationSecs": "1800",
+        "Workload": "oltp_insert",
+        "PackageName": "sysbench"
+      }
+    },
+    {
+      "Type": "SysbenchOLTPClientExecutor",
+      "Parameters": {
+        "Scenario": "oltp_insert_T96_TB4_REC100000",
+        "DatabaseName": "sbtest", 
+        "Role": "Client",
+        "Threads": "96",
+        "NumTables": "4",
+        "RecordCount": "100000",
+        "DurationSecs": "1800",
+        "Workload": "oltp_insert",
+        "PackageName": "sysbench"
+      }
+    },
+    {
+      "Type": "SysbenchOLTPClientExecutor",
+      "Parameters": {
+        "Scenario": "oltp_insert_T152_TB4_REC5000",
+        "DatabaseName": "sbtest", 
+        "Role": "Client",
+        "Threads": "152",
+        "NumTables": "4",
+        "RecordCount": "5000",
+        "DurationSecs": "1800",
+        "Workload": "oltp_insert",
+        "PackageName": "sysbench"
+      }
+    },
+    {
+      "Type": "SysbenchOLTPClientExecutor",
+      "Parameters": {
+        "Scenario": "oltp_insert_T152_TB4_REC10000",
+        "DatabaseName": "sbtest", 
+        "Role": "Client",
+        "Threads": "152",
+        "NumTables": "4",
+        "RecordCount": "10000",
+        "DurationSecs": "1800",
+        "Workload": "oltp_insert",
+        "PackageName": "sysbench"
+      }
+    },
+    {
+      "Type": "SysbenchOLTPClientExecutor",
+      "Parameters": {
+        "Scenario": "oltp_insert_T152_TB4_REC50000",
+        "DatabaseName": "sbtest", 
+        "Role": "Client",
+        "Threads": "152",
+        "NumTables": "4",
+        "RecordCount": "50000",
+        "DurationSecs": "1800",
+        "Workload": "oltp_insert",
+        "PackageName": "sysbench"
+      }
+    },
+    {
+      "Type": "SysbenchOLTPClientExecutor",
+      "Parameters": {
+        "Scenario": "oltp_insert_T152_TB4_REC100000",
+        "DatabaseName": "sbtest", 
+        "Role": "Client",
+        "Threads": "152",
+        "NumTables": "4",
+        "RecordCount": "100000",
+        "DurationSecs": "1800",
+        "Workload": "oltp_insert",
+        "PackageName": "sysbench"
+      }
+    }
+  ],
+  "Dependencies": [
+    {
+      "Type": "LinuxPackageInstallation",
+      "Parameters": {
+        "Scenario": "InstallRequiredLinuxPackages",
+        "Packages": "mysql-server, make, automake, libtool, pkg-config",
+        "Packages-Apt": "libaio-dev, libmysqlclient-dev, libssl-dev",
```

Comment:
Just to confirm, you need those libaio packages?

Reply:
yeah looks like it